src/dotnet/commands/dotnet-pack/PackageGenerator.cs

```diff
@@ -58,7 +59,7 @@ public bool BuildPackage(IEnumerable<ProjectContext> contexts, List<DiagnosticMe
 
             var packageOutputPath = Path.Combine(
                 ArtifactPathsCalculator.PackageOutputPath,
-                GetPackageName() + NuGet.Constants.PackageExtension);
+                GetPackageName() + NuGetConstants.PackageExtension);
```

Comment:
This change seems totally unrelated to the current work.  It really doesn't make sense to include it IMO.